src/copy-to-clipboard/styles.scss

```diff
@@ -0,0 +1,23 @@
+/*
+ Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ SPDX-License-Identifier: Apache-2.0
+*/
+
+@use '../internal/styles/tokens' as awsui;
+
+.root {
+  display: contents;
+}
+
+.inline-container {
+  display: inline-block;
+  word-break: break-all;
+
+  &-trigger {
+    display: inline-block;
+    margin-inline-end: awsui.$space-scaled-xxs;
+  }
+  &-text {
+    display: contents;
```

Comment:
How many of these styles would be needed after addressing the comment above about `<div>`?

Reply:
@pan-kot this question remains open. Did you try cleaning up these styles?

Reply:
Thanks, updated now